fastavro/six.py

```diff
@@ -65,10 +65,13 @@ def py2_utob(n, encoding=_encoding):
     _outenc = getattr(stdout, 'encoding', None) or _encoding
 
     def py2_json_dump(obj, indent):
+        kwargs = {}
```

Comment:
Unrelated to this PR, but I just noticed that `py3_json_dump` doesn't do any checking for `_HAS_UJSON`... it seems like it should, right? If you want to add that to this PR then that'd be great otherwise I'll make an issue and get back to it in the future

Reply:
Will do.